engine/apps/api/views/organization.py

```diff
@@ -46,7 +49,11 @@ def put(self, request):
 
 class GetTelegramVerificationCode(APIView):
     authentication_classes = (PluginAuthentication,)
-    permission_classes = (IsAuthenticated, IsAdmin)
+    permission_classes = (IsAuthenticated, RBACPermission)
+
+    rbac_permissions = {
+        "get": [RBACPermission.Permissions.OTHER_SETTINGS_WRITE],
```

Comment:
@matiasb wondering if it also makes sense to change this to `CHATOPS_UPDATE_SETTINGS`?